libs/designsystem/src/lib/directives/element-as-button/element-as-button.directive.spec.ts

```diff
@@ -24,8 +24,4 @@ describe('ElementAsButtonDirective', () => {
   it('should have cursor: pointer', () => {
     expect(spectator.element).toHaveComputedStyle({ cursor: 'pointer' });
   });
-
```

Comment:
Removed, as outline is no longer used as focus ring. General testing of the interaction states is carried out in  https://github.com/kirbydesign/designsystem/issues/2072